src/views/Farms/components/FarmTable/Actions/HarvestAction.tsx

```diff
@@ -42,11 +42,13 @@ const HarvestAction: React.FunctionComponent<FarmWithStakedValue> = ({ pid, user
     updateValue.current(earningsBusd)
   }, [earningsBusd, updateValue])
 
+  const tokenEarnedString = TranslateString(330, 'CAKE Earned', { asset: 'CAKE' }).split(' ')
+
   return (
     <ActionContainer>
       <ActionTitles>
-        <Title>CAKE </Title>
-        <Subtle>EARNED</Subtle>
+        <Title>{tokenEarnedString[0]} </Title>
+        <Subtle>{tokenEarnedString[1]}</Subtle>
```

Comment:
In this case it is okay to leave `CAKE` as is and just wrap `Earned` in the translate string
```
{TranslateString(999, 'Earned')}
```

Reply:
Done